build/azure-pipelines/build-product.yml

```diff
@@ -168,4 +168,15 @@ stages:
         inputs:
           targetPath: $(Build.ArtifactStagingDirectory)
           artifactName: SqlcmdRelease
-
+          
+      - task: GitHubRelease@1
+        inputs:
+          gitHubConnection: 'gosqlcmd_github'
+          repositoryName: '$(Build.Repository.Name)'
+          action: 'create'
+          target: '$(Build.SourceVersion)'
+          tagSource: 'userSpecifiedTag'
+          tag: '$(VERSION_TAG)'
+          assets: '$(Build.ArtifactStagingDirectory)/*'

```

Comment:
>assets: '$(Build.ArtifactStagingDirectory)/*' [](http://example.com/codeflow?start=10&length=45)

you can leave out the assets line, this task uploads all files in that folder by default

Reply:
Removed it. As the field was not automatically populated, I thought it may skip uploading. Thanks for clarifying.